packages/web-frontend/src/components/DateRangePicker/DatePickerDialog.js

```diff
@@ -135,7 +137,8 @@ export const DatePickerDialog = ({
     return setErrorMessage('');
   };
 
-  const minMomentDate = moment(MIN_DATE_PICKER_DATE);
+  const minMomentDate = min ? moment(min) : moment(MIN_DATE_PICKER_DATE);
```

Comment:
```suggestion
  const minMomentDate = min ? moment(min) : moment(DEFAULT_MIN_DATE);
```
Worth a rename now it's just a default

Reply:
Renamed